backend/dto/users.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable import/prefer-default-export */
+export class UserDTO {
+    id
+
+    username
+
+    nickname
+
+    profilePic
+
+    profileBanner
+
+    bio
+
+    followers
+
+    following
+
+    joinedDate
+
+    constructor(
+        id,
+        username,
+        nickname,
+        profilePic,
+        profileBanner,
+        bio,
+        followers,
+        following,
+        joinedDate
+    ) {
+        this.id = id
+        this.username = username
+        this.nickname = nickname
+        this.profilePic = profilePic
+        this.profileBanner = profileBanner
+        this.bio = bio
+        this.followers = followers
+        this.following = following
+        this.joinedDate = joinedDate
+    }
+
+    getUserObject() {
+        return {
+            id: this.id,
+            username: this.username,
+            nickname: this.nickname,
+            profilePic: this.profilePic,
+            profileBanner: this.profileBanner,
+            bio: this.bio,
+            followers: this.followers.length,
+            following: this.following.length,
+            joinedDate: this.joinedDate,
+        }
+    }
```

Comment:
This works, but a cleaner way to do it would be to make the userDTO be the final object that you send back to the front-end, so we could do something like `userDTO = convertToDto(user)` and then `res.status(200).send(userDTO)`